easybuild/easyconfigs/c/chewBBACA/chewBBACA-3.3.10-foss-2023a.eb

```
@@ -0,0 +1,78 @@
+easyblock = 'PythonBundle'
+
+name = 'chewBBACA'
+version = '3.3.10'
+
+homepage = 'https://github.com/B-UMMI/chewBBACA'
+description = """
+chewBBACA is a software suite for the creation and evaluation of core genome and
+whole genome MultiLocus Sequence Typing (cg/wgMLST) schemas and results.
+The "BBACA" stands for "BSR-Based Allele Calling Algorithm".
+BSR stands for BLAST Score Ratio as proposed by Rasko DA et al.
+The "chew" part adds extra coolness to the name and could be thought of as
+"Comprehensive and Highly Efficient Workflow"."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('scikit-build-core', '0.9.3'),
+    ('poetry', '1.5.1'),
+]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('Biopython', '1.83'),
+    ('plotly.py', '5.16.0'),
+    ('Cython', '3.0.8'),
+    ('BLAST+', '2.14.1'),
+    ('MAFFT', '7.520', '-with-extensions'),
+    ('FastTree', '2.1.11'),
+    ('archspec', '0.2.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+local_preinstallopts = (
+    "sed -i "
+    "-e 's/numpy~=1.24.3/numpy>=1.24.3/g' "
+    "-e 's/scipy~=1.10.1/scipy>=1.10.1/g' "
+    'pyproject.toml && '
+)
```

let's make this a bit more generic:

```suggestion
local_preinstallopts = "sed -i -e 's/numpy~=/numpy>=/g' -e 's/scipy~=/scipy>=/g' pyproject.toml && "